### ISSUE TITLE
operon copy and image

### ISSUE THREAD

**Issue by zuzzurro** [quodlibet/quodlibet#1935]
The copy command copies all the tags, but not the image. Would it be possible to add a copy-image command?

**Comment by lazka** [comment]
Sounds good

**Comment by zuzzurro** [comment]
Let me add a couple of comments:
1) I suggested to add a copy-image command because it seems that the images are not considered tags. Since the end result I will want to copy both the tags and the images I will end up calling the command twice. Adding a switch to the copy command would save the second invocation.
2) I noticed that running a copy command twice will duplicate all the tags. So in reality the invocation will end up being in order to achieve a simpe copy:

operon clear -a file.m4a
operon copy --ignore-errors file.flac file.m4a
operon copy-image file.flac file.m4a

that is not very optimal..

**Comment by lazka** [comment]
Yeah, not that optimal. I focused on keeping the interface simple instead of performance.

What about adding a new "clone" command which just tries to copy as much info as possible (ratings, images, tags etc..)

**Comment by zuzzurro** [comment]
I think that a generalized copy --all where the source and destination (see #1934) might be folder could be enough. That's the way the fileutils cp command behaves, after all.
